lib/FusionInventory/Agent/Task/Inventory/Win32/Softwares.pm

```diff
@@ -232,6 +232,29 @@ sub _getSoftwaresList {
             $software->{INSTALLDATE} = _dateFormat(_keyLastWriteDateString($data));
         }
 
+        #----- SQL Server -----
+        # Versions >= SQL Server 2008 (tested with 2008/R2/2012/2016) : "SQL Server xxxx Database Engine Services"
+        if ($software->{NAME} =~ /^(SQL Server.*)(\sDatabase Engine Services)/) {
+            my $sqlEditionValue = _getSqlInstancesList(
+                softwarename    => $software->{NAME},
+                softwareversion => $software->{VERSION}
+            );
+            if (defined($sqlEditionValue)) {
+                $software->{NAME} = "$1 " . $sqlEditionValue . "$2";
```

Comment:
why quote some variables and not others ?
$software->{NAME} = $1. " " . $sqlEditionValue . $2;
or 
$software->{NAME} = "$1 $sqlEditionValue$2";

Reply:
Old test, old debug I did. I'll formalized.